### DISCUSSION TITLE
hacknet

### DISCUSSION THREAD

**Discussion by luqiyihui** [Gamuxorg/bbs#234] [投递和请求游戏]
游戏名称：hacknet
游戏名称(英文)：hacknet
下载地址 :（如果有的话）：
网站中有hacknet，不过好像需要安装和登陆steam才能玩，能打包个不需要steam的64位的吗？谢谢！

**Comment by hyh19962008** [comment]
已更新，含LABYRINTH DLC
https://de.linuxgame.cn/Hacknet/Hacknet_amd64.appimage
